docs/install/linux_source.rst

```diff
@@ -12,8 +12,9 @@ PyNE has the following dependencies:
    #. `Fortran compiler <https://gcc.gnu.org/wiki/GFortran>`_
    #. `C++ compiler <https://gcc.gnu.org/>`_
    #. `CMake <http://www.cmake.org/>`_ (>= 2.8.5)
-   #. `NumPy <http://www.numpy.org/>`_ (>= 1.8.0)
+   #. `NumPy <http://www.numpy.org/>`_ (1.11.0)
    #. `SciPy <http://www.scipy.org/>`_
+   #. `Jinja2 <http://jinja.pocoo.org/docs/2.9/intro/>`_
```

Comment:
We should be careful not to overlist dependencies.  Can you hunt down exactly what needs jinja please?